src/lib/datepicker/datepicker-input.ts

```diff
@@ -280,7 +280,10 @@ export class MatDatepickerInput<D> implements AfterContentInit, ControlValueAcce
     return this._validator ? this._validator(c) : null;
   }
 
-  /** @deletion-target 7.0.0 Use `getConnectedOverlayOrigin` instead */
+  /**
+   * @deprecated
```

Comment:
@jelbourn do we want to mark these deprecated now or wait until 6.0.0?

Reply:
As early as possible is always good